examples/kernel_shap_wine_intro.ipynb

```diff
@@ -0,0 +1,1201 @@
+{
```

Comment:
<p>Output of <code>.fit</code>  using the new API seems to be missing the <code>name</code>  attribute.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Hmmm, I remember previous comments were saying that this should take care of itself?</p>

Reply:
<p>maybe setting it as <code>None</code>  in <code>DEFAULT_META_SHAP</code>  wasn't a good idea, I will remove that.</p>